service/src/integrationTest/resources/fixtures/case.json

```diff
@@ -7,6 +7,14 @@
         "orderType": [
           "EMERGENCY_PROTECTION_ORDER"
         ]
+      },
+      "tempC2": {
+        "c2upload": {
+          "document_url": "http://localhost/documents/85d97996-22a5-40d7-882e-3a382c8ae1b4",
+          "document_binary_url": "http://localhost/documents/85d97996-22a5-40d7-882e-3a382c8ae1b4/binary",
+          "document_filename": "file.pdf"
+        },
+        "c2UploadDescription": "test"
```

Comment:
Not sure if this should be in here or placed elsewhere :)

Reply:
Not the end of the world. We have two different sets of fixture files, and we chop and change which we use all over though... bad practice.